wsd/DocumentBroker.hpp

```diff
@@ -499,6 +513,7 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
     private:
         Status _status;
         bool _closeRequested; //< Owner-Termination flag.
+        bool _loaded; //< If the document ever loaded (check isLive to see if it still is).
```

Comment:
Again not std::atomic any more - is it not needed?

Reply:
Same as _closeRequested, _loaded is set when we get the "status: " message from Kit. We read it from the socket in poll and it's a set-only (no resetting). I don't see any thread contentions on these two variables, but I'll triple check to make sure we don't introduce any regressions.

Reply:
Restored atomic for the same reason as for _closeRequested.